tests/scripts/task_build_adreno_bins.sh

```diff
@@ -29,7 +29,7 @@ cd ${output_directory}
 cp ../cmake/config.cmake .
 
 echo set\(USE_MICRO OFF\) >> config.cmake
-echo set\(USE_CLML ON\) >> config.cmake
+echo set\(USE_CLML "${ADRENO_OPENCL}"\) >> config.cmake
```

Comment:
Is it not necessary to do this check (`[[ -z "${ADRENO_OPENCL}" ]] && CLML_PATH='ON' || CLML_PATH="${ADRENO_OPENCL}"`) here?

Reply:
We don't need this check in other script too as with in CI its always available as ```/adreno-opencl```.